apps/cbportal/base/cloudbroker__cloudspace/methodclass/cloudbroker_cloudspace.py

```diff
@@ -136,6 +136,42 @@ def removeIP(self, cloudspaceId, ipaddress, **kwargs):
         """
         return True
 
+    @auth(['level1', 'level2', 'level3'])
+    def getVFW(self, cloudspaceId, **kwargs):
+        """
+        Get VFW info
+        param:cloudspaceId id of the cloudspace
+        """
+        cloudspaceId = int(cloudspaceId)
+        if not self.models.cloudspace.exists(cloudspaceId):
+            raise exceptions.NotFound('Cloudspace with id %s not found' % (cloudspaceId))
+
+        cloudspace = self.models.cloudspace.get(cloudspaceId)
+        fwid = '%s_%s' % (cloudspace.gid, cloudspace.networkId)
+
+        if not self.vfwcl.virtualfirewall.exists(fwid):
+            raise exceptions.BadRequest('Can\'t get VFW of %s cloudspace' % (cloudspace.status))
+
+        network = self.vfwcl.virtualfirewall.get(fwid)
+        network_obj = network.dump()
+
+        if self.syscl.node.exists(network.nid):
+            network_obj['nodename'] = self.syscl.node.get(network.nid).name
+        else:
+            network_obj['nodename'] = str(network.nid)
+
+        network_obj['pubips'] = ', '.join(network.pubips)
```

Comment:
this shouldnt be transformed in a string in an api, perfectly fine to be returned as a list